contributors/devel/component-config-conventions.md

```diff
@@ -110,13 +106,14 @@ whether alternatives might be a better fit.
 
 ### Versioning Configuration
 
-We create configuration API groups per component that live in the source tree
-of the component. Each component has its own API group for configuration.
+We create configuration API groups per component that live in the source tree of
+the component. Each component has its own API group for configuration.
 Components will use the same API machinery that we use for other API groups.
 Configuration API serialization doesn't have the same performance requirements
 as other APIs so much of the codegen can be avoided (e.g. ugorji, generated
-conversions) and we can instead fallback to the reflection based
-implementations where they exist.
+conversions) and we can instead fallback to the reflection based implementations
+where they exist. Configuration API groups for component config should be named
+according to the scheme `<component>.config.k8s.io`.
```

Comment:
Do you want to split this sentence out as its own paragraph? I feel like it might get lost here.

Reply:
Done, and added some motivation for the naming.